free-2.md

```diff
@@ -11,34 +12,120 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The users I have have in mind are the users of latex who find it's syntax
+somewhat obtrusive. Right now, when people use LaTeX they need to type a lot of
+things, and sometimes the semantics are not fully clear.
+
+For example, to use an environment you have to do:
+
+```
+\begin{center}
+  <stuff to center>
+\end{center}
+```
+
+Notably many LaTeX users end up indenting the contents of an environment
+even though the language doesn't require it. Even those that choose not to
+agree that doing so make the code more readable.
+
+I would expand/restrict LaTeX to make it a whitespace sensitive language, which
+is just as full-featured as the original, but requires less typing and ties the
+layout of the code to the way that code is interpreted.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I see the problem as one involving the way users interface with the typesetting
+system, not as a problem with the typesetting system itself. By this I mean
+that while most people think LaTeX allows them to format things how they
+desire, this required syntax is often verbose and cumbersome.
+
+I think that a language is thus the perfect solution to this problem. Making a
+language is fundamentally about changing the way two entities interact.
+
+I think the particular problems that a language could solve are:
+   1. Making communication between the user and the computer more efficient
+      that (less typeing)
+   2. Aligning the way users tend to format their code (indentation blocks)
+      with the meaning of that code.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Almost everyone I talk to has opinions about LaTeX. Those who don't use it note
+the complexity of the syntax and the complexity of the build system. Those who
+do use it are excited about the typesetting power it provides to them. Those
+who use it a lot often complain about the verbosity of the code and the
+relative difficulty of finding bugs given the simplicity of the grammar of the
+language.
+
+I to use LaTeX, and between my own complaints and the complaints of my peers I
+would be excited to improve it in some way.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Typesetting for power users.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
-
+I imagine a syntax in which environments, macro definitions, and one-arguments
+macros use whitespace sensitive syntax, something like python:
```

Comment:
I think this is a great idea, mostly because all of those languages that don't require whitespace generally have style guides very strongly encouraging the use of whitespace.